src/style.css

```diff
@@ -1,3 +1,69 @@
+:root {
+  --blue-color: #00A7E1;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo img {
+  height: 40px;
+  position: absolute;
+  top: 10px;
+  left: 50px;
+}
```

Comment:
Avoid using tag names for styling. It's better to use class selectors to apply styles. Also, consider if the `position: absolute;` is necessary here, as it may lead to less maintainable code.